client/src/entrypoints/contrib/table_block/table.js

```diff
@@ -238,26 +233,23 @@ class TableInput {
     const container = document.createElement('div');
     container.innerHTML = `
       <div class="w-field__wrapper" data-field-wrapper>
-        <label class="w-field__label" for="${id}-handsontable-header">${this.strings['Row header']}</label>
-        <div class="w-field w-field--boolean_field w-field--checkbox_input" data-field>
-          <div class="w-field__help" id="${id}-handsontable-header-helptext" data-field-help>
-            <div class="help">${this.strings['Display the first row as a header.']}</div>
-          </div>
-          <div class="w-field__input" data-field-input>
-            <input type="checkbox" id="${id}-handsontable-header" name="handsontable-header" aria-describedby="${id}-handsontable-header-helptext" />
-          </div>
-        </div>
-      </div>
-      <div class="w-field__wrapper" data-field-wrapper>
-        <label class="w-field__label" for="${id}-handsontable-col-header">${this.strings['Column header']}</label>
-        <div class="w-field w-field--boolean_field w-field--checkbox_input" data-field>
-          <div class="w-field__help" id="${id}-handsontable-col-header-helptext" data-field-help>
-            <div class="help">${this.strings['Display the first column as a header.']}</div>
-          </div>
-          <div class="w-field__input" data-field-input>
-            <input type="checkbox" id="${id}-handsontable-col-header" name="handsontable-col-header" aria-describedby="${id}-handsontable-col-header-helptext" />
-          </div>
-        </div>
+        <label class="w-field__label" for="${id}-table-header-choice">${this.strings['Table headers']}</label>
+          <select id="${id}-table-header-choice" name="table-header-choice">
+            <option value="">Select a header option</option>
+            <option value="row">
+                ${this.strings['Display the first row as a header']}
+            </option>
+            <option value="column">
+                ${this.strings['Display the first column as a header']}
+            </option>
+            <option value="both">
+                ${this.strings['Display the first row AND first column as headers']}
+            </option>
+            <option value="neither">
+                ${this.strings['No headers']}
+            </option>
+          </select>
+        <p class="help">${this.strings['Which cells should be displayed as headers?']}</p>
```

Comment:
While we are here, can we associate this help text with the input - aria-describedby?

Reply:
im not good with the django templates yet but may i ask why with the input - aria-describedby? 🤔 
shouldn't the below changes be enough?
```suggestion
        <div class="w-field__help" aria-describedby="${id}-handsontable-col-caption-helptext" data-field-help>
            <p class="help">${this.strings['Which cells should be displayed as headers?']}</p>
          </div>
```